lib/model/tokens.dart

```diff
@@ -469,13 +469,13 @@ class CustomIntBuffer {
   }
 
   void put(int value) {
-    if (_list.length >= maxSize) list.removeAt(0);
-    _list.add(value);
+    if (list.length >= maxSize) list.removeAt(0);
```

Comment:
The field is still private, here the getter is called instead because lazy initialization is done for the variable. I.e., this prevents the list from being null.